src/Admin/AdminConsole/Controllers/OrganizationsController.cs

```diff
@@ -243,6 +243,20 @@ public async Task<IActionResult> Delete(Guid id)
         return RedirectToAction("Index");
     }
 
+    [HttpPost]
+    [ValidateAntiForgeryToken]
+    [RequirePermission(Permission.Org_Delete)]
+    public async Task<IActionResult> DeleteInitiation(Guid id, OrganizationInitiateDeleteModel model)
+    {
+        var organization = await _organizationRepository.GetByIdAsync(id);
```

Comment:
After adding the data annotations to the model we need to enforce their validation here:
```suggestion
        if (!ModelState.IsValid)
        {
            return RedirectToAction("Edit", new { id });
        }

        var organization = await _organizationRepository.GetByIdAsync(id);
```